ccdproc/core.py

```diff
@@ -571,6 +572,88 @@ def transform_image(ccd, transform_func, **kwargs):
     return nccd
 
 
+@log_to_metadata
+def wcs_project(ccd, target_wcs, target_shape=None, order='bilinear'):
+    """
+    Given a CCDData image with WCS, project it onto a target WCS and
+    return the reprojected data as a new CCDData image.
+
+    Any mask, flags, weight, or uncertainty are ignored in doing the
```

Comment:
Is this a copy/paste error? I think at least the mask is not ignored during "reprojection" (instead of ``combination``)